pom.xml

```diff
@@ -33,10 +33,10 @@
 
  <repositories>
     <!-- Including for fontawesome -->
-    <repository>
+ <repository>
       <id>fontawesomefx-repo</id>
       <name>FontAwesome Repository</name>
-      <url>https://bintray.com/jerady/maven/FontAwesomeFX</url>
+      <url>https://dl.bintray.com/jerady/maven</url>
     </repository>
   </repositories>
```

Comment:
It seems like maven will first try fontawesome repo and then central?
E.g. logs:
..
Downloading from fontawesomefx-repo: https://dl.bintray.com/jerady/maven/org/hibernate/common/hibernate-commons-annotations/5.0.4.Final/hibernate-commons-annotations-5.0.4.Final.pom
Downloading from central: https://repo.maven.apache.org/maven2/org/hibernate/common/hibernate-commons-annotations/5.0.4.Final/hibernate-commons-annotations-5.0.4.Final.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/hibernate/common/hibernate-commons-annotations/5.0.4.Final/hibernate-commons-annotations-5.0.4.Final.pom (1.9 kB at 16 kB/s)
Downloading from fontawesomefx-repo: https://dl.bintray.com/jerady/maven/org/springframework/data/spring-data-jpa/2.1.8.RELEASE/spring-data-jpa-2.1.8.RELEASE.pom
Downloading from central: https://repo.maven.apache.org/maven2/org/springframework/data/spring-data-jpa/2.1.8.RELEASE/spring-data-jpa-2.1.8.RELEASE.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/springframework/data/spring-data-jpa/2.1.8.RELEASE/spring-data-jpa-2.1.8.RELEASE.pom (13 kB at 73 kB/s)
...

I used "mvn dependency:purge-local-repository" and then mvn install to get the log above